cms/djangoapps/contentstore/tests/utils.py

```diff
@@ -125,158 +123,6 @@ def save_course(self):
     DRAFT_VIDEO = 'draft_video'
     LOCKED_ASSET_KEY = AssetKey.from_string('/c4x/edX/toy/asset/sample_static.html')
 
-    def import_and_populate_course(self):
```

Comment:
Just as a sanity check: It's okay to remove this because there's already an equivalent testing split mongo specifically, right?

Reply:
Yes we have similar test for split mongo, so I think we can remove this test.